docs/graphene-workload-tutorial/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Writing a Python workload to run with Graphene SGX
+
+## Changes to customize the template
+
+For this tutorial let us consider a check for palindrome as the business logic for our workload.
+Bring up the docker container from current directory with the name of your workload as an environment variable. For example -
+```
+WORKLOAD_NAME=palindrome docker-compose up
+```
+This will prepare your the setup for building your workload. You just need to add the business logic.
+
+### Source code changes in template.py
+
+Uncomment all code blocks in the function `_palindrome()`. This function holds the core business logic.
```

Comment:
Let's keep a file xxx.py.template and create a file xxx.py form the template.

Reply:
Updated